schemavalidator/util.py

```diff
@@ -0,0 +1,58 @@
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def log_error_table(validator, document, schema, level=logging.DEBUG):
+    errors = collect_errors(validator, document, schema)
+    table = errors_to_table(errors)
+    if len(table) > 0:
+        logger.log(level, format_table(table, ['schema path', 'instance path',
```

Comment:
Put the `logger.log` in schemavalidator.py, otherwise the log event is registered from utils.py, which may be confusing.